Source/inv.cpp

```diff
@@ -1808,18 +1808,21 @@ bool TryInvPut()
 	if (numitems >= MAXITEMS)
 		return false;
 
-	direction dir = GetDirection(plr[myplr].position.tile, { cursmx, cursmy });
-	if (CanPut(plr[myplr].position.tile.x + offset_x[dir], plr[myplr].position.tile.y + offset_y[dir])) {
+	Direction dir = GetDirection(plr[myplr].position.tile, { cursmx, cursmy });
+	Point position = plr[myplr].position.tile + dir;
+	if (CanPut(position.x, position.y)) {
 		return true;
 	}
 
-	direction dirLeft = left[dir];
-	if (CanPut(plr[myplr].position.tile.x + offset_x[dirLeft], plr[myplr].position.tile.y + offset_y[dirLeft])) {
+	Direction dirLeft = left[dir];
+	position = plr[myplr].position.tile + dirLeft;

```

Comment:
Do we even need the variable at this point?
```suggestion
	position = plr[myplr].position.tile + left[dir];
```